gennav/planners/graph_search/astar.py

```diff
@@ -27,27 +33,27 @@ def astar(graph, start, end, heuristic={}):
             graph(dict): Dictionary representing the graph,
                     where keys are the nodes and the
                     value is a list of all neighbouring nodes
-            start(tuple): Tuple representing key corresponding to the start point
```

Comment:
pretty minor but can you add spaces between param names and the (type) annotation